packages/dd-trace/test/appsec/sdk/set_user.spec.js

```diff
@@ -0,0 +1,133 @@
+'use strict'
+
+const proxyquire = require('proxyquire')
+const agent = require('../../plugins/agent')
+const tracer = require('../../../../../index')
+const getPort = require('get-port')
+const axios = require('axios')
+
+describe('set_user', () => {
+  describe('Internal API', () => {
+    const tracer = {}
+
+    let rootSpan, getRootSpan, setUser, log
```

Comment:
huge nitpick but at this point, why not do it :rofl: 
```suggestion
    let rootSpan, getRootSpan, log, setUser
```